react/react1/week3/todo-app3/src/components/Context.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState, createContext } from 'react';
+
+export const UserContext = createContext({});
+
+export function UserProvider({ children }) {
+  const [input, setInput] = useState('');
+  const [list, setList] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState(false);
+
+  return (
+    <UserContext.Provider
+      value={{
```

Comment:
you don't need all these values, only input and setInput as far as I can see.

Reply:
She needs all of them. 
- input's state in UserList.js
- others in ResultData.js